2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -849,6 +832,142 @@ $bold("Scenario: ")$
+    [FM-5.2.2], [Obbligatorio], [L'utente visualizza un errore per l'inserimento di dimensioni che non permettono all'oggetto di essere inserito nell'ambiente.], [UC-5.2],
+
+    [FD-6],[Desiderabile], [L'utente deve poter richiedere il caricamento dei dati da database], [UC-6],
+    [FO-], [Opzionale], [L'utente deve poter configurare i parametri di connessione al database], [],
+    [FO-], [Opzionale], [L'utente deve poter indicare il nome del database], [],
+    [FO-], [Opzionale], [L'utente deve poter indicare il nome utente per la connessione al database], [],
+    [FO-], [Opzionale], [L'utente deve poter indicare la password per la connessione al database], [],
+    [FO-], [Opzionale], [L'utente deve poter indicare l'indirizzo del database], [],
+    [FO-], [Opzionale], [L'utente deve poter indicare la porta del database], [],
+    [FD-], [Desiderabile], [L'utente deve poter testare la connessione al database], [],
+    [FD-], [Desiderabile], [L'utente visualizza un errore se i dati contenuti nel database non sono conformi], [],
+    [FD-], [Desiderabile], [L'utente visualizza un errore se i dati contenuti nel database sono errati], [],
+
+    [FM], [Obbligatorio], [L'utente deve poter richiedere lo spostamento di un prodotto da un bin ad un altro], [],
+    [FM], [Obbligatorio], [L'utente deve poter richiedere lo spostamento di un prodotto da un bin ad un altro indicando le coordinate del bin di destinazione], [],
+    [FM], [Obbligatorio], [L'utente deve poter richiedere lo spostamento di un prodotto da un bin ad un altro tramite _drag and drop_], [],
+    [FM], [Obbligatorio], [Il sistema deve interrogare una API RESTful per accertare che lo spostamento sia lecito], [],
+    [FD], [Desiderabile], [Il sistema deve evidenziare il bin di partenza per rendere evidente la richiesta di spostamento], [],
+    [FD], [Desiderabile], [Il sistema deve evidenziare il bin di destinazione per rendere evidente la richiesta di spostamento], [],
+
+    [FM], [Obbligatorio], [L'utente deve poter visualizzare le informazioni di un bin selezionato], [],
+    [FM], [Obbligatorio], [L'utente deve poter visualizzare le informazioni del prodotto contenuto in un bin selezionato], [],
+    [FM], [Obbligatorio], [L'utente deve poter visualizzare le informazioni di uno scaffale selezionato], [],
+
+    [FD], [Desiderabile], [L'utente deve poter ricercare un prodotto], [],
+    [FD], [Desiderabile], [L'utente deve poter ricercare un prodotto per ID], [],
+    [FD], [Desiderabile], [L'utente deve poter ricercare un prodotto per nome], [],
+    [FD], [Desiderabile], [L'utente deve poter ricercare uno scaffale], [],
+    [FD], [Desiderabile], [Il sistema deve fornire la lista dei risultati di ricerca], [],
+    [FD], [Desiderabile], [Il sistema deve evidenziare i risultati di ricerca], [],
```

Comment:
questa parte finale della tabella sembra molto work in progress, mancano i riferimenti finali e a volte i codici indentificativi, in più sono presenti commenti come quello di riga 963.

Reply:
la tabella verrà ultimata in fase di tracciamento dei requisiti (per cui è stata aperta una task Jira), dove verranno aggiunti anche i relativi codici degli use case